openmdao/core/tests/test_expl_comp.py

```diff
@@ -166,6 +166,118 @@ def test_compute_and_list(self):
         with self.assertRaisesRegex(RuntimeError, msg):
             prob.model.list_outputs(explicit=False, implicit=False)
 
+    def test_list_inputs_outputs_with_parallel_comps(self):
+        class TestComp(om.ExplicitComponent):
+
+            def initialize(self):
+                self.options['distributed'] = False
+
```

Comment:
To reproduce (and test) reported issue,, you need an MPI test with 3 procs.

Reply:
I actually need 4 to make reproduce the the issue. Fixed